parquet-benchmarks/run.sh

```diff
@@ -21,10 +21,10 @@
 SCRIPT_PATH=$( cd "$(dirname "$0")" ; pwd -P )
 
 echo "Starting WRITE benchmarks"
-java -jar ${SCRIPT_PATH}/target/parquet-benchmarks.jar p*Write* "$@"
+java -jar ${SCRIPT_PATH}/target/parquet-benchmarks.jar org.apache.parquet.benchmarks.WriteBenchmarks "$@"
```

Comment:
With this change, `NestedNullWritingBenchmarks` won't be executed in this benchmark suite, but I think that's fine. @gszadovszky do you agree?

Reply:
I agree. When this script was created only `ReadBenchmarks` and `WriteBenchmarks` were exist. However, it might make sense to have a more descriptive name (e.g. `runReadWrite.sh`).

Reply:
I combined the two existing run scripts into one with some very limited functionality to run predefined "suites" by keyword, but also to run all benchmarks.

What do you think?